<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputItemWidget</class>
 <widget class="QWidget" name="OutputItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="MessageLabel" name="messageLabel_" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="FileInfoLabel" name="fileLabel_">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>3</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="expandFileInfoTb_">
           <property name="toolTip">
            <string>Expand file info label</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="copyPathTb_">
           <property name="toolTip">
            <string>Copy file path</string>
           </property>
           <property name="text">
            <string>Copy file path</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/copy_path.svg</normaloff>:/viewer/copy_path.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="saveFileAsTb_">
           <property name="toolTip">
            <string>Save a local copy of the current file to disk</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/filesaveas.svg</normaloff>:/viewer/filesaveas.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="fetchInfoTb_">
           <property name="toolTip">
            <string>Additional information about the fetched output file</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/grey_info.svg</normaloff>:/viewer/grey_info.svg</iconset>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toStartTb_">
           <property name="toolTip">
            <string>Go to start of document</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/arrow_top.svg</normaloff>:/viewer/arrow_top.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toEndTb_">
           <property name="toolTip">
            <string>Go to end of document</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/arrow_bottom.svg</normaloff>:/viewer/arrow_bottom.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toLineStartTb_">
           <property name="toolTip">
            <string>Go to start of line</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/arrow_left.svg</normaloff>:/viewer/arrow_left.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toLineEndTb_">
           <property name="toolTip">
            <string>Go to end of line</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/arrow_right.svg</normaloff>:/viewer/arrow_right.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="fontSizeUpTb_">
           <property name="toolTip">
            <string>Increase font size in text browser &lt;br&gt;&lt;code&gt;Ctrl++ or Ctrl+wheel&lt;/code&gt;</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/fontsize_up.svg</normaloff>:/viewer/fontsize_up.svg</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl++</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="fontSizeDownTb_">
           <property name="toolTip">
            <string>Descrease font size in text browser &lt;br&gt;&lt;code&gt;Ctrl+- or Ctrl+wheel&lt;/code&gt;</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/fontsize_down.svg</normaloff>:/viewer/fontsize_down.svg</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+-</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="wordWrapTb_">
           <property name="toolTip">
            <string>Text wrap</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/wordwrap.svg</normaloff>:/viewer/wordwrap.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="searchTb_">
           <property name="toolTip">
            <string>Show search bar (CTRL-F)</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/search_decor.svg</normaloff>:/viewer/search_decor.svg</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+F</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="gotoLineTb_">
           <property name="toolTip">
            <string>Goto line number (CTRL-L)</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/images/goto_line.svg</normaloff>:/viewer/images/goto_line.svg</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+L</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="lineNumberTb_">
           <property name="toolTip">
            <string>Show line numbers</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/line_num.svg</normaloff>:/viewer/line_num.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="filterTb_">
           <property name="toolTip">
            <string>Toggle  to &lt;b&gt;show&lt;/b&gt; text filter bar</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/filter_decor.svg</normaloff>:/viewer/filter_decor.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="filterOptionTb_">
           <property name="toolTip">
            <string>Text filter menu</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/down_arrow.svg</normaloff>:/viewer/down_arrow.svg</iconset>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="reloadTb_">
           <property name="toolTip">
            <string>Reload the selected output file</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/sync.svg</normaloff>:/viewer/sync.svg</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+R</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="moreActionTb_">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/cogwheel.svg</normaloff>:/viewer/cogwheel.svg</iconset>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="MessageLabel" name="warnLabel_" native="true"/>
       </item>
       <item>
        <widget class="OutputBrowser" name="browser_" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dirWidget_" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
         <item>
          <widget class="DirInfoLabel" name="dirLabel_">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="dirReloadTb_">
           <property name="toolTip">
            <string>Reload directory listing</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="viewer.qrc">
             <normaloff>:/viewer/sync_black.svg</normaloff>:/viewer/sync_black.svg</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="dirView_">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="MessageLabel" name="dirMessageLabel_" native="true"/>
   </item>
  </layout>
  <action name="actionSaveFileAs_">
   <property name="icon">
    <iconset resource="viewer.qrc">
     <normaloff>:/viewer/filesaveas.svg</normaloff>:/viewer/filesaveas.svg</iconset>
   </property>
   <property name="text">
    <string>Save file as ...</string>
   </property>
   <property name="toolTip">
    <string>Save a local copy of the current task to disk</string>
   </property>
  </action>
  <action name="actionGotoLine_">
   <property name="icon">
    <iconset resource="viewer.qrc">
     <normaloff>:/viewer/images/goto_line.svg</normaloff>:/viewer/images/goto_line.svg</iconset>
   </property>
   <property name="text">
    <string>Go to line ...</string>
   </property>
   <property name="toolTip">
    <string>Go to line number</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageLabel</class>
   <extends>QWidget</extends>
   <header>MessageLabel.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileInfoLabel</class>
   <extends>QLabel</extends>
   <header>FileInfoLabel.hpp</header>
  </customwidget>
  <customwidget>
   <class>DirInfoLabel</class>
   <extends>QLabel</extends>
   <header>FileInfoLabel.hpp</header>
  </customwidget>
  <customwidget>
   <class>OutputBrowser</class>
   <extends>QWidget</extends>
   <header>OutputBrowser.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="viewer.qrc"/>
 </resources>
 <connections/>
</ui>
